examples/runtime_error/assert_eq_3.nbt

```diff
@@ -0,0 +1,2 @@
+# fail approximate equality test with epsilon in units that are different from either comparand
+assert_eq(2m, 201cm, 0.1cm to mm)
```

Comment:
Why not
```suggestion
assert_eq(2m, 201cm, 1mm)
```

Reply:
No strong reason, but this makes it easier to vary the units of the comparison and error message without having to change the digits of epsilon. For example, starting with `0.1cm` changing to `1mm` is a little bit harder than just adding `to mm`.